### PR TITLE
samples: zigbee: add zigbee tag for samples

### PR DESCRIPTION
**Pull request by nordic-piks**
Extend list of paths used to check if zigbee
samples should be executed.

### CODE REVIEW

**Comment by nordic-piks** [comment]
The idea behind this change is to allow automatic detection of PRs not related to given code area.
In this case, there is defined tag (e.g. zigbee) and files/folder with source code related to that area.
This list of files/folders is then compared to files changed in PR. If particular PR does not change them,
tests related to specified tag should not be executed - will be excluded.

**Comment by nordic-piks** [comment]
Updated after evaluation of:
https://github.com/nrfconnect/sdk-nrf/blob/main/.github/test-spec.yml#L222-L241
https://github.com/nrfconnect/sdk-nrfxlib/blob/main/.github/test-spec.yml#L47-L58
https://github.com/nrfconnect/sdk-zephyr/blob/main/.github/test-spec.yml#L94-L98

Please note that the content is similar and syntax is different - the root path is NCS, all dependencies - from west modules - are stated at one place, tests and samples should not be included here, they are handled separately (but they will be build if changed)

**Comment by nordic-piks** [comment]
DNM until NCS 2.7

**Review by tejlmand — APPROVED** [review]

**Comment by nordic-piks** [comment]
Just rebase and added path for tests and samples.

File: scripts/ci/tags.yaml [hunk 1]
```diff
@@ -154,3 +154,35 @@ crypto:
     - zephyr/include/drivers/serial/
     - zephyr/modules/mbedtls/
     - modules/crypto/
+
+zigbee:
+  files:
```
Comment (mariusz-nordicsemi) [inline scripts/ci/tags.yaml:159]: What about following files included in test-spec.yaml?
- nrf/applications/zigbee_weather_station
- nrf/samples/nrf5340/multiprotocol_rpmsg

**Review by mariusz-nordicsemi — CHANGES_REQUESTED** [review]

File: samples/zigbee/ncp/sample.yaml
[the remark this answers is not in the document]
Comment (nordic-piks) [reply samples/zigbee/ncp/sample.yaml:1]: Added, thanks ;)

File: scripts/ci/tags.yaml (hunk 1 above)
Comment (nordic-piks) [reply scripts/ci/tags.yaml:159]: Added.

**Review by mariusz-nordicsemi — APPROVED** [review]

**Pull request merged**